src/Admin/AbstractAdmin.php

```diff
@@ -284,7 +284,7 @@
     /**
      * The current request object.
      *
-     * @var \Symfony\Component\HttpFoundation\Request
+     * @var ?\Symfony\Component\HttpFoundation\Request
```

Comment:
This is valid PHPDoc?

Reply:
I tried using phpdocumentor on this and it does weird things like `\?\Datetime`, I'm going to use `|null` instead